.github/workflows/backups.yml

```diff
@@ -20,8 +20,10 @@ jobs:
       - id: set-matrix
         run: |
           matrix=$(curl https://nf-co.re/pipeline_names.json)
+          # Extract the pipeline array from the JSON structure
+          matrix=$(echo $matrix | jq '.pipeline')
           if [ -n "${{ inputs.pipeline }}" ] && [ "${{ inputs.pipeline }}" != "all" ]; then
-            matrix=$(echo $matrix | jq --arg pipeline "${{ inputs.pipeline }}" '[.[] | select(.name == $pipeline)]')
+            matrix=$(echo $matrix | jq --arg pipeline "${{ inputs.pipeline }}" '[.[] | select(. == $pipeline)]')
```

Comment:
I tested this and it returns an empty string, but not sure how to fix it, what is the expected result? the name of the pipeline?

Reply:
false alarm, I was doing it wrong, it returns a list with the name of the pipeline 🎉